lightning/restore/restore.go

```diff
@@ -592,7 +594,54 @@ func (rc *RestoreController) restoreTables(ctx context.Context) error {
 				return errors.Trace(err)
 			}
 			if cp.Status <= CheckpointStatusMaxInvalid {
-				return errors.Errorf("Checkpoint for %s has invalid status: %d", tableName, cp.Status)
+				allInvalidCheckpoints[tableName] = cp.Status
+			}
+		}
+	}
+
+	if len(allInvalidCheckpoints) != 0 {
+		logger := log.L()
+		logger.Error(
+			"TiDB Lightning has failed last time. To prevent data loss, this run will stop now. Please resolve errors first",
+			zap.Int("count", len(allInvalidCheckpoints)),
+		)
+
+		for tableName, status := range allInvalidCheckpoints {
+			failedStep := status * 10
+			var action strings.Builder
+			action.WriteString("./tidb-lightning-ctl --checkpoint-errors-")
+			switch failedStep {
+			case CheckpointStatusAlteredAutoInc, CheckpointStatusAnalyzed:
+				action.WriteString("ignore")
+			default:
+				action.WriteString("destroy")
+			}
+			action.WriteString("='")
+			action.WriteString(tableName)
+			action.WriteString("' --config=...")
```

Comment:
Didn't find `--config` in doc or cmd/tidb-lightning-ctl/main, what is this used for?